components/molecules/LinkedImage/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { FC } from "react";
+import classNames from "classnames";
+import Image, { ImageProps } from "next/image";
+import { LinkProps } from "next/link";
+import { type IconKey } from "@rubin-epo/epo-react-lib/IconComposer";
+import MasonryImage from "@rubin-epo/epo-react-lib/MasonryImage";
+import styles from "./styles.module.css";
+
+interface LinkedImageProps {
+  className?: string;
+  title?: string;
+  image: ImageProps;
+  link: LinkProps;
+  icon?: IconKey;
+}
+
+const LinkedImage: FC<LinkedImageProps> = ({
+  title,
+  image: { className: imageClassName, ...image },
+  link,
+  icon,
+  className,
+}) => {
+  return (
+    <MasonryImage
```

Comment:
generic kind of image-with-a-link and description that can be used in media asides and gallery masonry